pastoral-care.md

```diff
@@ -1,7 +1,8 @@
-# Pastoral Care 
-Officials are arguing over whether the novel coronavirus has reached the level of pandemic, as outbreaks continue to spread globally. Meanwhile, religious leaders still must care for people—many of whom are feeling a rise in anxiety verging on panic.
+# चरगाही की देखभाल 
+अधिकारी इस बात पर बहस कर रहे हैं कि क्या उपन्यास कोरोनावायरस महामारी के स्तर तक पहुंच गया है, क्योंकि वैश्विक स्तर पर इसका प्रकोप जारी है।
+इस बीच, धार्मिक नेताओं को अभी भी लोगों की देखभाल करनी चाहिए - जिनमें से कई घबराहट में चिंता की स्थिति में वृद्धि महसूस कर रहे हैं।
 
-### How can ministers, chaplains, counselors, and educators accompany people pastorally through this valley of anxiety, fear, and death? Here are 10 brief guidelines.
+### चिंता, भय, और मौत की घाटी के माध्यम से मंत्री, पादरी, परामर्शदाता और शिक्षक कैसे लोगों के साथ रह सकते हैं? यहां 10 संक्षिप्त दिशानिर्देश दिए गए हैं।
```

Comment:
This seems out of place

Reply:
I think it should be

चिंता, भय, और मौत के दर  से मंत्री, पादरी, परामर्शदाता और शिक्षक कैसे लोगों के साथ रह सकते हैं?

Reply:
I think डर would be fine